handlers/apps.go

```diff
@@ -0,0 +1,77 @@
+package handlers
+
+
+import (
+	"fmt"
+	"github.com/knative-scout/app-api/models"
+	"go.mongodb.org/mongo-driver/bson"
+	"net/http"
+	"strings"
+)
+
+// AppsHandler is used search apps and return result
+// in case of an empty query, it returns all the apps in the database
+
+type AppsHandler struct {
+	BaseHandler
+}
+
+// ServeHTTP implements http.Handler
+func (h AppsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+
+	//gets all the optional parameters passed in the URL
+	vars := r.URL.Query()
+	query := vars.Get("query")
+	tags := vars.Get("tags")
+	categories := vars.Get("categories")
+
+	resp := getDataFromDB(query, tags, categories, h)
+
+	h.RespondJSON(w, http.StatusOK, resp)
```

Comment:
Can you make it return data according to the spec? https://github.com/knative-scout/app-api/blob/master/DESIGN.md#search-apps

Should return an object with an `apps` can which should then holds the apps you've found

Reply:
okay